MiniSim/Menu.swift

```diff
@@ -116,9 +130,7 @@ class Menu: NSMenu {
     private func assignKeyEquivalent(devices: [NSMenuItem]) {
         for (index, item) in devices.enumerated() {
             if index > maxKeyEquivalent {
-                DispatchQueue.main.async {
-                    item.keyEquivalent = ""
-                }
+                item.keyEquivalent = ""
```

Comment:
Coming from iOS UI calls should always be on the main queue.I believe it was crashing, because this function wasn't called in the main queue. Main reason – `getDevices` is executing code in `DispatchQueue.global(qos: .userInitiated)` and this chains down to this function. That's what I've noticed when I was debugging app. 

But, thanks for pointing out! Looks like Swift Concurrency is not applied correctly here. I would like to update PR and remove last commit.